katcp/resource_client.py

```diff
@@ -1473,13 +1473,9 @@ def call_in_ioloop(self, fn, args, kwargs, timeout=None):
             return future.result(timeout)
         except TimeoutError:
             raise
-        except KeyboardInterrupt:
```

Comment:
@martinslabber That seems like a philosophical point, since the effect is the same. Perhaps tell me over coffee :)